File: sdks/csharp/services/Service.cs
// file: sdks/csharp/services/Service.cs
// version: 1.0.0
// guid: 2beebf59-6769-48de-8748-1b240967ecb0

using Sdks.Csharp.Client;

namespace Sdks.Csharp.Services;

/// <summary>
/// Example service wrapper.
/// </summary>
public class Service
{
    private readonly Client _client;

    public Service(Client client)
    {
        _client = client;
    }

    public void Call()
    {
        // TODO: implement RPC call
    }

    // TODO: add additional methods
    // TODO: handle errors
    // TODO: include retries
    // TODO: add documentation
}

File: sdks/java/services/Service.java
// file: sdks/java/services/Service.java
// version: 1.0.0
// guid: 41f8d6a4-54ec-4752-beb1-c6c2b35f0728

package sdks.java.services;

import sdks.java.client.Client;

/**
 * Example service wrapper.
 */
public class Service {
  private final Client client;

  public Service(Client client) {
    this.client = client;
  }

  public void call() {
    // TODO: implement RPC call
  }

  // TODO: add additional methods
  // TODO: map errors to exceptions
  // TODO: implement retries
  // TODO: document usage
}
